docs/source/usage/plugins/saxs.rst

```diff
@@ -0,0 +1,76 @@
+.. _usage-plugins-saxs:
+
+SAXS
+---------
+
+This plugin calculates Small Angle X-ray Scattering (SAXS) patterns from particle positions.
+
+.. math::
+
+   I({\bf q}) &= \frac{F({\bf q})F({\bf q})^*}{N} \\
+   F({\bf q}) &= \sum^{N}_{j=1}f_j\exp({\bf q \cdot r}_j)
```

Comment:
```suggestion
   F({\bf q}) &= \sum^{N}_{j=1}f_j\exp(i{\bf q \cdot r}_j)
```
I think the 'i' was missing here.

Reply:
@pordyna That's true. Thanks for correction.